upstream-community-operators/couchdb-operator/couchdb-operator.package.yaml

```diff
@@ -1,7 +1,9 @@
 channels:
 - currentCSV: couchdb-operator.v1.0.13
   name: stable
-- currentCSV: couchdb-operator.v1.0.13
+- currentCSV: couchdb-operator.v1.3.0
   name: beta
-defaultChannel: stable
+- currentCSV: couchdb-operator.v1.3.0
+  name: 1.3
+defaultChannel: 1.3
```

Comment:
The version should be 1.3.0
instead of 1.3
This is the reason the prepare operator metadata failed.